app/core/src/main/java/com/fsck/k9/notification/CertificateErrorNotificationController.kt

```diff
@@ -6,6 +6,7 @@ import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 import com.fsck.k9.Account
 
+@Deprecated("Remove once all usages have been removed for the new account edit feature")
```

Comment:
These notifications are created when a certificate error is encountered while checking messages in the background. We currently have no plans to remove this error reporting mechanism.

Reply:
Removed